shared/gcp_jupyterlab_shared/handlers.py

```diff
@@ -31,11 +32,14 @@
 METADATA_HEADER = {'Metadata-Flavor': 'Google'}
 SCOPE = ('https://www.googleapis.com/auth/cloud-platform',)
 FRAMEWORK_ENV_VAR = 'ENV_VERSION_FILE_PATH'
-
+SSL_OPTIONS = {"ssl_version": ssl.PROTOCOL_TLSv1}
```

Comment:
https://docs.python.org/3.6/library/ssl.html#ssl.PROTOCOL_TLSv1 suggests that this is deprecated. We have had this library supporting Scheduler since last year on Notebooks VMs and haven't had any issues.

I'm somewhat reluctant to introduce it to fix a local development issue if it could potentially affect production behavior.